routingv8/routes.go

```diff
@@ -0,0 +1,38 @@
+package routingv8
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+)
+
+// Routes returns routes.
+// The required parameters for this resource is an origin location and a destination location.
```

Comment:
We now take in a transportMode. If we don't provide a default one, all of the fields in the request has to be set. Perhaps we can then remove this comment about which fields are mandatory.

I think I would be fine by throwing an error if an invalid transportMode was provided.

(Also see my other comment suggesting using the TransportMode enum instead of string)